lib/screens/category_meals_screen.dart

```diff
@@ -0,0 +1,41 @@
+import 'package:flutter/material.dart';
+import '../widgets/meal_item.dart';
+
+import '../dummy_data.dart';
+
+class CategoryMealsScreen extends StatelessWidget {
+  static const routeName = '/category-meals';
+  // final String categoryId;
+  // final String categoryTitle;
+
+  // CategoryMealsScreen(this.categoryId, this.categoryTitle);
```

Comment:
do you still need this?

Reply:
I'm using it for reference for now

Reply:
if the succeeding pull requests make no use of those, please remove them.